wp_api_integration_tests/tests/test_cookie_authentication.rs

```diff
@@ -0,0 +1,66 @@
+use wp_api::{WpApiClient, WpAuthentication, WpLoginCredentials};
+
+use wp_api_integration_tests::*;
+
+use serial_test::serial;
+
+#[tokio::test]
+#[serial]
+async fn test_cookie_authentication() {
+    let login = WpLoginCredentials {
+        username: "test@example.com".to_string(),
```

Comment:
```suggestion
        username: TestCredentials::instance().admin_username.to_string(),
```

This suggestion applies to other 2 tests as well.